wellregistry/registry/models.py

```diff
@@ -29,16 +133,17 @@ class Registry(models.Model):
     We could refactor later.
 
     """
-    # these will become lookups with                    foreign keys
-    agency_cd = models.CharField(max_length=20)       # AGENCY_LOV.AGENCY_CD
-    well_depth_units = models.IntegerField()          # UNITS_DIM.UNIT_ID
-    alt_datum_cd = models.CharField(max_length=10)    # ALT_DATUM_DIM.ADATUM_CD
-    alt_units = models.IntegerField()                 # UNITS_DIM.UNIT_ID
-    horz_datum = models.CharField(max_length=10)      # HORZ_DATUM_DIM.HDATUM_CD
-    nat_aquifer_cd = models.CharField(max_length=10)  # NAT_AQFR.NAT_AQFR_CD
-    country_cd = models.ForeignKey(CountryLookup, on_delete=models.CASCADE)       # COUNTRY.COUNTRY_CD
-    state_cd = models.CharField(max_length=2)         # STATE.STATE_CD and STATE.COUNTRY_CD
-    county_cd = models.CharField(max_length=3)        # COUNTY.COUNTY_CD and COUNTY.STATE_CD and COUNTY.COUNTRY_CD
+    # these columns use foreign keys
+    agency_cd = models.ForeignKey(AgencyLovLookup, on_delete=models.PROTECT) # AGENCY_LOV.AGENCY_CD
+    well_depth_units = models.ForeignKey(UnitsDimLookup, related_name='+',
+                                         on_delete=models.PROTECT) # UNITS_DIM.UNIT_ID
+    alt_datum_cd = models.ForeignKey(AltDatumDimLookup, on_delete=models.PROTECT) # ALT_DATUM_DIM.ADATUM_CD
+    alt_units = models.ForeignKey(UnitsDimLookup, on_delete=models.PROTECT) # UNITS_DIM.UNIT_ID
+    horz_datum = models.ForeignKey(HorzDatumDimLookup, on_delete=models.PROTECT) # HORZ_DATUM_DIM.HDATUM_CD
+    nat_aquifer_cd = models.ForeignKey(NatAqfrLookup, on_delete=models.PROTECT)  # NAT_AQFR.NAT_AQFR_CD
+    country_cd = models.ForeignKey(CountryLookup, on_delete=models.PROTECT)  # COUNTRY.COUNTRY_CD
+    state_cd = models.ForeignKey(StateLookup, on_delete=models.PROTECT) # STATE.STATE_CD and STATE.COUNTRY_CD
+    county_cd = models.ForeignKey(CountyLookup, on_delete=models.PROTECT) # COUNTY.{country_cd,state_cd,county_cd'}
```

Comment:
I placed these together during analysis of the initial admin registry story so it would be easy to find them for this story. when we are comfortable with this update then they could be intermixed together as related groups. For example for altitude value: alt_va, alt_datum_cd, and alt_units near each other in the file. This does not have to be done for this PR nor ever if there is no concern from others.